protos/sigstore_bundle.proto

```diff
@@ -57,9 +57,38 @@ message TimestampVerificationData {
 // the key identifier, it MUST match the `keyid` field of the signature the
 // extension is attached to.
 message VerificationMaterial {
+        // The key material for verification purposes.
+        //
+        // This allows key material to be conveyed in one of three forms:
+        //
+        // 1. An unspecified public key identifier, for retrieving a key
+        //    from an out-of-band mechanism (such as a keyring);
+        //
+        // 2. A sequence of one or more X.509 certificates, of which the first member
+        //    MUST be a leaf certificate conveying the signing key.
+        //
+        //    Signers MUST NOT include root CA certificates in bundles, and SHOULD NOT
+        //    include intermediate CA certificates that appear in an independent root of trust
+        //    (such as the Public Good Instance's trusted root).
+        //
+        //    Verifiers MUST validate the chain carefully to ensure that it chains up
+        //    to a CA certificate that they independently trust. Verifiers SHOULD
+        //    handle old or non-complying bundles that have additional intermediate and/or
```

Comment:
Should we define "handle" more precisely? Something about ignoring root CA certificates if present? As for intermediates, that's currently up to the client. Cosign is the outlier in that it'll read in any intermediates provided. Maybe we mention something about clients providing a separate path for PGI vs private instances?

Reply:
Yeah, good catch -- I'll make it explicit that verifiers SHOULD handle these by ignoring any superfluous CA certificates (either by simply ignoring them entirely, or by ensuring that their chain building process treats them as untrusted for the purposes of verification). If I understand correctly, that'll be consistent with what `cosign` does 🙂